third_party/nvfuser/csrc/python_frontend/python_bindings.cpp

```diff
@@ -1474,6 +1474,32 @@ void initNvFuserPythonBindings(PyObject* module) {
       py::arg("arg"),
       py::arg("dtype"),
       py::return_value_policy::reference);
+  nvf_ops.def(
```

Comment:
nitpick: we are adding new API here, let's bump `version.txt`!

https://github.com/csarofeen/pytorch/blob/devel/third_party/nvfuser/version.txt.
Note that it's a simple text file, so make sure you double check the version.txt content prior to merging you PR, in case other PRs have bumped that in devel already.

Sorry for the inconvenience.